test/FunctionsV2/OutOfProcTests.cs

```diff
@@ -122,5 +138,209 @@ public async Task CallHttpActionOrchestrationWithManagedIdentity()
+                    managementUrls,
+                    "terminatePostUri",
+                    new[] { idPlaceholder });
+
+                Uri rewindPostUri = commonUriValidation(
+                    managementUrls,
+                    "rewindPostUri",
+                    new[] { idPlaceholder });
+
+                Uri purgeHistoryDeleteUri = commonUriValidation(
+                    managementUrls,
+                    "purgeHistoryDeleteUri",
+                    new[] { idPlaceholder });
+
+                commonUriValidation(outerJson, "baseUrl", new string[0]);
+
+                Assert.True(outerJson.TryGetValue("requiredQueryStringParameters", out JToken requiredQueryStringParameters));
+                Assert.Equal(testNotificationUrl.Query.Trim('?'), (string)requiredQueryStringParameters);
+
+                Assert.True(outerJson.TryGetValue("rpcBaseUrl", out JToken rpcBaseUrl));
+
+                if (localRcpEnabled)
+                {
+                    Assert.True(Uri.TryCreate((string)rpcBaseUrl, UriKind.Absolute, out Uri rpcBaseUri));
+                    Assert.True(rpcBaseUri.IsLoopback);
+                    Assert.Equal("http", rpcBaseUri.Scheme);
+                    Assert.Equal("/durabletask/", rpcBaseUri.AbsolutePath);
+                }
+                else
+                {
+                    Assert.Equal(JTokenType.Null, rpcBaseUrl.Type);
+                }
+            }
+        }
+
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [InlineData(null, true)]
+        [InlineData("node", true)]
+        [InlineData("java", true)]
+        [InlineData("powershell", true)]
+        [InlineData("python", true)]
+        [InlineData("dotnet", false)]
+        public async Task TestLocalRcpEndpointRuntimeVersion(string runtimeVersion, bool enabledExpected)
+        {
+            INameResolver nameResolver = new SimpleNameResolver(
+                new Dictionary<string, string>
+                {
+                    { "FUNCTIONS_WORKER_RUNTIME", runtimeVersion },
+                });
+
+            using (ITestHost host = TestHelpers.GetJobHost(
+                this.loggerProvider,
+                nameof(this.TestLocalRcpEndpointRuntimeVersion),
+                enableExtendedSessions: false,
+                localRpcEndpointEnabled: null /* use FUNCTIONS_WORKER_RUNTIME to decide */,
+                nameResolver: nameResolver))
+            {
+                await host.StartAsync();
+
+                // Check to see whether the local RPC endpoint has been opened
+                IPGlobalProperties ipGlobalProperties = IPGlobalProperties.GetIPGlobalProperties();
+                IPEndPoint[] endpoints = ipGlobalProperties.GetActiveTcpListeners();
+
+                const string LocalRcpAddress = "127.0.0.1:17071";
+                if (enabledExpected)
+                {
+                    Assert.Contains(LocalRcpAddress, endpoints.Select(ep => ep.ToString()));
```

Comment:
To be a bit more robust, we should check that this endpoint is not already open before the test. If it is, is it possible for a test to be marked as inconclusive?

Reply:
I suppose, but if it was already opened then the test would fail when we do `host.StartAsync()`.

Reply:
It also looks like xunit doesn't allow you to mark a test as inconclusive from code.

Reply:
Fair enough. Let's just leave it as is.